### PR TITLE
Fix loss functions doc

### PR DESCRIPTION
**Pull request by ZhuBaohe**
Correct docstring display error on web page caused by my previous PR

### CODE REVIEW

**Review by zou3519 — APPROVED** [review]

**Comment by zou3519** [comment]
You should fix the lint @ZhuBaohe: https://travis-ci.org/pytorch/pytorch/jobs/510830807

also, it would be nice if you posted some screenshots of how it looks after your changes

**Review by zou3519 — CHANGES_REQUESTED** [review]
see comment

**Comment by ZhuBaohe** [comment]
I have fixed Travis CI fail.

**Comment by soumith** [comment]
thank you @ZhuBaohe !

**Review by soumith — APPROVED** [review]
